banyan/resource_service_db.go

```diff
@@ -71,6 +71,12 @@ func DbSchema() map[string]*schema.Schema {
 			Required:    true,
 			Description: "The external-facing network address for this service; ex. website.example.com",
 		},
+		"suppress_device_trust_verification": {
+			Type:        schema.TypeBool,
+			Description: "If device trust verification is suppressed, mobile devices will not be able to pass the device trust check.",
```

Comment:
Did the text for these Descriptions come from our documentation site? I am not sure they make sense.

Reply:
@yoshiotu no i would need proper text